Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -243,6 +243,43 @@ lemma pow_lt_pow_succ (ha : 1 < a) : a ^ n < a ^ n.succ := by
 lemma pow_lt_pow_right₀ (ha : 1 < a) (hmn : m < n) : a ^ m < a ^ n := by
   induction' hmn with n _ ih; exacts [pow_lt_pow_succ ha, lt_trans ih (pow_lt_pow_succ ha)]
 
+lemma zpow_lt_zpow_of_lt_one₀ (ha₀ : 0 < a) (ha₁ : a < 1) {m n : ℤ} (hmn : m < n) : a ^ n < a ^ m :=
+  zpow_right_strictAnti₀ ha₀ ha₁ hmn
+
+lemma le_of_zpow_le_zpow₀ (ha : 1 < a) {n m : ℤ} (hmn : a ^ m ≤ a ^ n) :
+    m ≤ n := by
+  contrapose! hmn
+  exact zpow_lt_zpow_right₀ ha hmn
+
+lemma le_of_zpow_le_zpow_of_lt_one₀ (ha₀ : 0 < a) (ha₁ : a < 1) {m n : ℤ}
+    (hmn : a ^ m ≤ a ^ n) :
+    n ≤ m := by
+  contrapose! hmn
+  exact zpow_lt_zpow_of_lt_one₀ ha₀ ha₁ hmn
```

Comment:
This can be made an iff, should be called `zpow_lt_zpow_iff_right_of_lt_one₀` and should be proved and located similarly to `zpow_lt_zpow_iff_right₀`

Reply:
done

Reply:
@YaelDillies Could you explain what the "right" in the name refers to?

Reply:
It refers to the fact that we are considering the binary operator `^` as a function of its right argument, namely `a ^ ·`. `_left` would refer to the function `· ^ n` instead